openpype/hosts/maya/plugins/publish/extract_camera_mayaScene.py

```diff
@@ -133,8 +133,7 @@ def process(self, instance):
         # get cameras
         members = cmds.ls(instance.data['setMembers'], leaf=True, shapes=True,
                           long=True, dag=True)
-        cameras = cmds.ls(members, leaf=True, shapes=True, long=True,
-                          dag=True, type="camera")
+        cameras = cmds.ls(members, type="camera")
```

Comment:
Please keep `long=True`

Reply:
If I keep `long=True` it will be the same issue as before. If there's an other flag than the `type="camera"` the command will not work properly.

Reply:
`long=True` should work fine. It's the combination of `dag`, `leaf` or `shapes` causing the actual issue. I believe it's `dag`.